### PR TITLE
Add V2 contracts

### PR DESCRIPTION
**Pull request by maks-ivanov**
**NOTE**

1. The protocol is usually listed within 24 hours of merging the PR
2. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
3. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
4. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
5. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---

### CODE REVIEW

**Comment by g1nt0ki** [comment]
hi @maks-ivanov , can you fix the merge conflicts

File: projects/algofi/index.js [hunk 1]
```diff
@@ -278,7 +372,7 @@ module.exports = {
     algorand: {
         tvl: sdk.util.sumChainTvls([supply, dex]),
         borrowed,
-        staking
+        staking: stakingV1 + stakingV2
```
Comment (g1nt0ki) [inline projects/algofi/index.js:375]: this should be like tvl

`staking: sdk.util.sumChainTvls([stakingV1, stakingV2]),`

**Review by g1nt0ki — CHANGES_REQUESTED** [review]

**Comment by llamatester** [comment]
The adapter at projects/algofi exports TVL: 
        
 
 
    algorand                  89.08 M
    algorand-borrowed         37.71 M
    algorand-staking          12.47 M
    borrowed                  37.71 M
    staking                   12.47 M
    
    total                    89.08 M

**Comment by llamatester** [comment]
Error while running adapter at projects/trustswap: 
        
 
  
    
    
    Error: call revert exception [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (method="balanceOf(address)", data="0x", errorArgs=null, errorName=null, errorSignature=null, reason=null, code=CALL_EXCEPTION, version=abi/5.6.4)
    Truncated error stack:
        at Interface.decodeFunctionResult (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@ethersproject/abi/lib/interface.js:388:23)

File: projects/algofi/index.js (hunk 1 above)
Comment (maks-ivanov) [reply projects/algofi/index.js]: done

**Comment by g1nt0ki** [comment]
can you remove the `.DS_Store` file from trustswap folder

**Pull request merged**